docs/.vuepress/config/theme-config.js

```diff
@@ -1,5 +1,5 @@
 const themeConfig = {
-  logo: '/assets/logo-beta.png',
+  logo: '/assets/logo.png',
```

Comment:
```suggestion
  logo: '/assets/logo-beta.png',
```

This shouldn't have been committed 😅 Modifying it so we keep the beta logo on beta docs :-)